docs/docs/component-css.md

```diff
@@ -2,7 +2,192 @@
 title: Component CSS
 ---
 
-This is a stub. Help our community expand it.
+### Component CSS
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Gatsby has a wealth of options available for styling components. This guide will explore one very popular method: CSS Modules. If you would like to see other CSS options, check out the bottom of the page: [Other CSS Options](#other-css-options).
+
+### CSS Modules
+
+Quoting from
+[the CSS Module homepage](https://github.com/css-modules/css-modules):
+
+> A **CSS Module** is a CSS file in which all class names and animation names
+> are scoped locally by default.
+
+CSS Modules are very popular as they let you write CSS like normal but with a lot
+more safety. The tool automatically makes class and animation names unique so
+you don't have to worry about selector name collisions.
+
+CSS Modules are highly recommended for those new to building with Gatsby (and
+React in general).
+
+Gatsby works out of the box with CSS Modules.
+
+This guide will outline the steps necessary to build a page using CSS Modules.
+
```

Comment:
Hm, I know that this issue specifically instructed you to copy and paste from the tutorial. Now that I see it, I'm realizing that linking to the tutorial at this point will be the easiest to maintain down the road. Apologies for the change! Will you replace this copied/pasted content below this comment with a very basic code example and then a link to the tutorial? It will make the guide fairly short, which is fine!

Reply:
@amberleyromo what kind of code example would make sense in this guide?

Reply:
I will wait for a response

Reply:
I would say pull out the initial example: